### DISCUSSION TITLE
wallet connection for mobile in react

### DISCUSSION THREAD

**Discussion by sriharsh1104** [phantom/docs#210] [Help & Support] [answered]
### Summary

i want to connect phantom wallet for solana for website plus for my mobile chrome browser but i can't able to connect any example for react js no react-native or else 

### Steps to Reproduce

upper ka solution 

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hi @sriharsh1104,

Thanks for writing in!

[Deeplinks](https://docs.phantom.app/phantom-deeplinks/deeplinks-ios-and-android) are best for dapp connections outside of the Phantom Mobile Dapp Browser.
